pom.xml

```
@@ -2,7 +2,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>de.fraunhofer.iem</groupId>
     <artifactId>pathexpression</artifactId>
-    <version>1.0.4</version>
+    <version>1.0.5</version>
```

```suggestion
    <version>1.0.6</version>
```
1.0.5 already deployed! we should update the workflow to update: patchversion++ + -SNAPSHOT with a release